### ISSUE TITLE
Issue with german localization and SCGI

### ISSUE THREAD

**Issue by Shadow4711** [backuppc/backuppc#111]
Hello,

I have the problem that after switching to SCGI umlauts in the german localization are not displayed correctly.

Feel free to request more details if needed.

![umlauts](https://cloud.githubusercontent.com/assets/28490473/25775158/40628432-329f-11e7-9d32-7327ac549fa1.PNG)

Regards
Shadow4711

**Comment by craigbarratt** [comment]
Likely an incorrect http header.  Can you tell me the http headers you see with scgi vs cgi?

**Comment by Shadow4711** [comment]
You are right:

With SGCI: `Content-Type: text/html; charset=iso-8859-1`
With CGI: `Content-Type: text/html; charset=utf-8`

Any ideas where this comes from?

**Comment by craigbarratt** [comment]
That explains what you are seeing, but I don't know why the charset isn't utf-8.  In both the SCGI and CGI cases, the header is printed in `lib/BackupPC/CGI/Lib.pm`:
```
print $Cgi->header(-charset => "utf-8");
```

**Comment by Shadow4711** [comment]
Could you please check if this is a general issue when using BackupPC with SCGI? I want to doublecheck if this could be possibly caused by my setup.

**Comment by craigbarratt** [comment]
I'm traveling this week in Asia and I won't be able to check that out until this weekend.

**Comment by craigbarratt** [comment]
I've confirmed on my 4.1.2 setup the header response is:
```
Content-Type:text/html; charset=utf-8
```
So we need to figure out why your SCGI setup uses iso-8859-1.

**Comment by craigbarratt** [comment]
Quick update - while the charset is correct, my SCGI web page in German also displays the umlauts incorrectly.  So I am able to replicate your problem.  Somewhere SCGI must be doing some sort of charset conversion. I'll work on fixing it.

**Comment by craigbarratt** [comment]
I push a fix 964f964.
